clang/runtime/dpct-rt/include/device.hpp

```diff
@@ -138,7 +138,7 @@ class device_info {
       _name[255] = '\0';
     }
   }
-  void set_max_work_item_sizes(const sycl::id<3> max_work_item_sizes) {
+  void set_max_work_item_sizes(const sycl::range<3> max_work_item_sizes) {
```

Comment:
1. please add a new interface for backward compatibility. 
2. try to unify _max_work_item_sizes and _max_work_item_sizes_i.

Reply:
To avoid sync between _max_work_item_sizes and _max_work_item_sizes_i,  just keep one is enough. 
construct another if need in use point.